### ISSUE TITLE
Usage of P82_at_some_time_within

### ISSUE THREAD

**Issue by tpluscode** [linked-art/linked.art#505]
We are triplifying data with defined time spans such as people birth date only as the year of birth and death

According to LA, would this be a correct output?

```json
{
   "@context":"https://linked.art/ns/v1/linked-art.json",
   "id":"http://example.com/Person",
   "died":{
      "type":"Death",
      "timespan":{
         "type":"TimeSpan",
         "at_some_time_within":{
            "type":"xsd:gYear",
            "@value":"2000"
         }
      }
   },
   "born":{
      "type":"Birth",
      "timespan":{
         "type":"TimeSpan",
         "at_some_time_within":{
            "type":"xsd:gYear",
            "@value":"1950"
         }
      }
   }
}
```

**Comment by azaroth42** [comment]
No, for spans we use the P82a/b pattern of begin_of_the_begin and end_of_the_end.
See the model info here: https://linked.art/model/base/#time-span-details 

There was a relatively recent discussion about this in the SIG: https://cidoc-crm.org/Issue/ID-417-beginofthebegin-endoftheend-is-excluded-from-time-range which culminated in the answer that the end_of_the_end is included in the range, not excluded. Meaning the year 1950 would be begin_of_the_begin: 1950-01-01T00:00:00 and end_of_the_end 1950-12-31T23:59:59, and not and end_of_the_end of 1951-01-01T00:00:00 (which is much easier to calculate!)
